modules/nf-core/chromap/chromap/tests/main.nf.test

```diff
@@ -142,4 +114,33 @@ nextflow_process {
         }
 
     }
+
+    test("sarscov2 - single_end - stub") {
+
+        options "-stub"
+
+        when {
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test', single_end:true ], // meta map
+                    [file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/fastq/test_1.fastq.gz', checkIfExists: true)]
+                ])
+                input[1] = Channel.of([[:], [file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)]])
+                input[2] = CHROMAP_INDEX.out.index
```

Comment:
Setup should be done in stub too, no?

Reply:
I moved it out of all tests because the tests kept failing and it was always the same setup command. Is that wrong?